docs/layers/tile-layer.md

```diff
@@ -94,32 +94,112 @@ new deck.TileLayer({});
 
 ## Properties
 
-It implements geospatial-specific `tileToBoundingBox` and `getTileIndices` functions as props and also inherits all other [BaseTileLayer](/docs/api-reference/base-tile-layer.md) properties
+### Data Options
 
-### Render Options
-
-##### `getTileData` (Function,  optional)
+##### `getTileData` (Function)
 
 `getTileData` given x, y, z indices of the tile, returns the tile data or a Promise that resolves to the tile data.
 
-- Default: `getTileData: ({x, y, z}) => Promise.resolve(null)`
+- Default: `tile => Promise.resolve(null)`
+
+The `tile` argument contains the following fields:
+
+- `x` (Number) - x index of the tile
+- `y` (Number) - y index of the tile
+- `z` (Number) - z index of the tile
+- `bbox` (Object) - bounding box of the tile, see `tileToBoundingBox`.
+
+By default, the `TileLayer` loads tiles defined by [the OSM tile index](https://wiki.openstreetmap.org/wiki/Slippy_map_tilenames). You may override this by implementing `getTileIndices`.
+
+
+##### `maxZoom` (Number|Null, optional)
+
+Use tiles from this level when over-zoomed.
+
+- Default: `null`
+
+
+##### `minZoom` (Number, optional)
+
+Hide tiles when under-zoomed.
+
+- Default: 0
+
+
+##### `maxCacheSize` (Number|Null, optional)
+
+The maximum cache size for a tile layer. If not defined, it is calculated using the number of tiles in the current viewport times multiplied by `5`.
+
+- Default: `null`
+
+
+##### `strategy` (Enum, optional)
```

Comment:
Not a strong opinion here, but perhaps `refinementStrategy`, `tileRefinementStrategy` `tileSelectionStrategy` etc might be a bit more clear than just `strategy`?